### PR TITLE
feat: added support for podDisruptionBudget

### PR DESCRIPTION
**Pull request by simonhargos**
This PR adds support for declaring a Pod Disruption Budget and resolves https://github.com/dexidp/helm-charts/issues/22

### CODE REVIEW

**Review by sagikazarmark — CHANGES_REQUESTED** [review]
Thanks @simonhargos 

Can you please follow the template and values based on this example:

- https://github.com/sagikazarmark/helm-charts/blob/master/charts/kube-secrets-init/templates/pdb.yaml
- https://github.com/sagikazarmark/helm-charts/blob/master/charts/kube-secrets-init/values.yaml#L164-L173

You will also need to bump the chart version and update the artifacthub changelog.

Thanks!

**Comment by simonhargos** [comment]
> Thanks @simonhargos
> 
> Can you please follow the template and values based on this example:
> 
> * https://github.com/sagikazarmark/helm-charts/blob/master/charts/kube-secrets-init/templates/pdb.yaml
> * https://github.com/sagikazarmark/helm-charts/blob/master/charts/kube-secrets-init/values.yaml#L164-L173
> 
> You will also need to bump the chart version and update the artifacthub changelog.
> 
> Thanks!

Ok sure, let me take a look.

**Comment by sagikazarmark** [comment]
Please also make sure to sign your commits.

File: charts/dex/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  {{- with .Values.podDisruptionBudget.minAvailable }}
+  minAvailable: {{ . }}
+  {{- end }}
+  {{- with .Values.podDisruptionBudget.maxUnavailable }}
+  maxUnavailable: {{ . }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "dex.selectorLabels" . | nindent 6 }}
+{{- end }}
```
Comment (sagikazarmark) [inline charts/dex/templates/poddisruptionbudget.yaml:22]: Please adhere editorconfig settings (missing newline)

**Review by sagikazarmark — CHANGES_REQUESTED** [review]

File: charts/dex/templates/poddisruptionbudget.yaml (hunk 1 above)
Comment (simonhargos) [reply charts/dex/templates/poddisruptionbudget.yaml]: Hopefully that's fixed.

**Review by sagikazarmark — APPROVED** [review]
LGTM, thanks @simonhargos

**Comment by ChrisCooney1** [comment]
Could this be merged ASAP please? We could really use this on our team!

**Pull request merged**